public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/onCall/onCall.ts

```diff
@@ -0,0 +1,19 @@
+import { Receiver } from '../../../../../../../plugins/datasource/alertmanager/types';
```

Comment:
just curious why do you prefer relative imports rather than absolute?

Reply:
no idea why this import is like this 😆...fixing it!

Reply:
btw we use https://github.com/grafana/oncall/blob/dev/grafana-plugin/tools/eslint-rules/no-relative-import-paths.js rule to restrict using relative imports